server/app/services/geo/esri/EsriClient.java

```diff
@@ -180,9 +75,9 @@ public CompletionStage<Optional<AddressSuggestionGroup>> getAddressSuggestions(A
         .thenApply(
             (maybeJson) -> {
               if (maybeJson.isEmpty()) {
-                logger.error(
-                    "EsriClient.fetchAddressSuggestions JSON response is empty. Called by"
-                        + " EsriClient.getAddressSuggestions. Address = {}",
+                this.logger.error(
+                    "RealEsriClient.fetchAddressSuggestions JSON response is empty. Called by"
+                        + " RealEsriClient.getAddressSuggestions. Address = {}",
```

Comment:
I think the logged message should still use `EsriClient` and not `RealEsriClient` since it's within the EsriClient. If this is specific to the Real client we should make it abstract and move this into there.

Reply:
Thanks for catching this. I think I had an iteration where this was just in the Real one and didn't change this back.